Ginger/GingerCoreNET/Drivers/CoreDrivers/Web/Selenium/SeleniumDriver.cs

```diff
@@ -673,7 +680,7 @@ public override void StartDriver()
                             SetBrowserLogLevel(EDOpts);
                             //EDOpts.AddAdditionalEdgeOption("UseChromium", true);
                             //EDOpts.UseChromium = true;
-                            EDOpts.UnhandledPromptBehavior = UnhandledPromptBehavior.Default;
+                            EDOpts.UnhandledPromptBehavior = OpenQA.Selenium.UnhandledPromptBehavior.Default;
```

Comment:
Please use SetUnhandledPromptBehavior(options); function for all browsers to set the default behavior

Reply:
Implemented all the suggested modifications.